lib/loklak.rb

```diff
@@ -24,4 +24,31 @@ def self.status()
       {}
     end
   end
+
+  def self.search(query=nil, since_date=nil, until_date=nil, from_user=nil, count=nil)
+    method_path = 'search.json?'
```

Comment:
Remove the `?` from here and put it before `query=` That makes more sense than having a query endpoint with a `?` trailing it.